src/rpc/misc.cpp

```diff
@@ -215,21 +221,23 @@ UniValue z_validateaddress(const UniValue& params, bool fHelp)
     if (fHelp || params.size() != 1)
         throw runtime_error(
             "z_validateaddress \"zaddr\"\n"
-            "\nReturn information about the given z address.\n"
+            "\nVerify an zaddress is valid.\n"
```

Comment:
```suggestion
            "\nReturn information about the given zaddress.\n"
```
Original wording was OK.